cspm/rn/prisma-cloud-compute-release-information/features-introduced-in-compute-january-2023.adoc

```diff
@@ -0,0 +1,682 @@
+* GET, api/vVERSION/audits/runtime/container/download
+* GET, api/vVERSION/audits/runtime/container/timeslice
+* GET, api/vVERSION/audits/runtime/file-integrity
+* GET, api/vVERSION/audits/runtime/file-integrity/download
+* GET, api/vVERSION/audits/runtime/host
+* GET, api/vVERSION/audits/runtime/host/download
+* GET, api/vVERSION/audits/runtime/host/timeslice
+* GET, api/vVERSION/audits/runtime/log-inspection
+* GET, api/vVERSION/audits/runtime/log-inspection/download
+* GET, api/vVERSION/audits/runtime/serverless
+* GET, api/vVERSION/audits/runtime/serverless/download
+* GET, api/vVERSION/audits/runtime/serverless/timeslice
+* GET, api/vVERSION/audits/trust
+* GET, api/vVERSION/audits/trust/download
+
+//GH#36823 (PCC-727)
+|*Immediate Image Scanning*
+|Introduces a body parameter, onDemandScan, that triggers an on-demand image scan without interrupting the current or ongoing scan for the following API:
+* POST, api/vVERSION/registry/scan
+
+NOTE: The image's registry must be predefined in the registry settings.
+
+//GH#36867 (PCEE and SaaS) 
+|*Severity Level Based Report for Vulnerabilities*
+|Introduces a query parameter normalizedSeverity for host, images, registry, VMs, and serverless APIs to report vulnerabilities based on severity level.
+
+You can use the following APIs to report vulnerabilities based on the normalized severity:
+
+* GET, api/vVERSION/images
+* GET, api/vVERSION/images/download
+* GET, api/vVERSION/hosts
+* GET, api/vVERSION/hosts/download
+* GET, api/vVERSION/serverless
+* GET, api/vVERSION/serverless/download
+* GET, api/vVERSION/registry
+* GET, api/vVERSION/registry/download
+* GET, api/vVERSION/vms,
+* GET, api/vVERSION/vms/download
+
+//GH#37375 
+|*Supports Viewing 250 Reports or Entries Per Page*
+|The query parameter limit now supports a page size of 250 entries or reports. The default value is 50 entries or reports per page.
+
+For example: Use the following way to retrieve the first 250 reports with a limit query parameter for an API endpoint /hosts:
+[userinput]
+----
+$ curl -k \
+  -u <USER> \
+  -H 'Content-Type: application/json' \
+  -X GET \
+  ‘https://<CONSOLE>/api/v<VERSION>/hosts?limit=250&offset=0’
+----
+
+//GH# 37465 (PCSUP-7446) 
+|*Support for More Registry Entries*
+|You can now add or edit up to 19,999 registry entries by using the following API:
+* POST, api/vVERSION/settings/registry
+* PUT, api/vVERSION/settings/registry
+
+|*DISA STIG Scan Findings and Justifications*
+|Every https://docs.paloaltonetworks.com/prisma/prisma-cloud/prisma-cloud-compute-edition-public-sector/Release_Findings[release], we perform an SCAP scan of the Prisma Cloud Compute Console and Defender images. The process is based upon the U.S. Air Force’s Platform 1 "Repo One" OpenSCAP scan of the Prisma Cloud Compute images. We compare our scan results to IronBank’s latest approved UBI8-minimal scan findings. Any discrepancies are addressed or justified.
+
+|===
+
+[#id-addressed-issues]
+=== Addressed Issues
+
```

Comment:
@divyabhushan missing this bug for the addressed issues, it was also fixed for update 1 https://redlock.atlassian.net/browse/CWP-42634
CVSS3 score wasn't displayed for Harbor 2.5 and up (there was a change on Harbor's side), so we fixed it to handle their change.

Reply:
I added CWP-42634 to this RN. Please add it to the PCCE addressed issues too, @divyabhushan

Reply:
Thanks @Pubs-MV. I added the addressed issue in PCCE [here](https://github.com/PaloAltoNetworks/prisma-cloud-docs/pull/767/commits/6517d3c4a9423b6e1eea656c5b57efc68579e284].